web/src/main/java/org/cbioportal/genome_nexus/web/param/PtmFilter.java

```diff
@@ -0,0 +1,20 @@
+package org.cbioportal.genome_nexus.web.param;
+
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.List;
+
+// TODO add other filter params (uniprot, position, type, pubmed id, etc.) when needed
```

Comment:
Not sure if we want to spend time adding more query options at this point. Currently all we need is to query by transcript id.